src/modules/dashboard/components/ColonyDecisions/DecisionPreview.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Heading from '~core/Heading';
+import Button from '~core/Button';
+import Tag from '~core/Tag';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import { useUser, useLoggedInUser, useColonyFromNameQuery } from '~data/index';
+
+import DetailsWidget from './DetailsWidget/DetailsWidget';
+
+import styles from './DecisionPreview.css';
+
+const MSG = defineMessages({
+  type: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.type',
+    defaultMessage: `Type`,
+  },
+  team: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.team',
+    defaultMessage: `Team`,
+  },
+  author: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.author',
+    defaultMessage: `Author`,
+  },
+  edit: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.edit',
+    defaultMessage: `Edit`,
+  },
+  publish: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.publish',
+    defaultMessage: `Publish`,
+  },
+  // this is mock data for the decision title until a later PR is created to retrieve this from local storage
+  title: {
```

Comment:
I did it like this because I wasn't sure what technique we use to represent mock data. My assumption is that this will be replaced when the title and description is retrieved from local storage. What is the proper way to represent mock data?

Reply:
You just declare it as a variable with the data structure that you think we will use in the future. In this case it would be something like:
```
const data = {
  title: 'blabla',
  description: 'blabla',
  domainId: 2,
  <anything else that's needed>
}